index.tt

```diff
@@ -6,6 +6,11 @@
 
 <link rel="alternate" type="application/rss+xml" title="RSS" href="/news-rss.xml" />
 
+<div class="alert alert-info">
+  This years <a href="https://2020.nixcon.org/">NixCon</a> will take place <b>online</b> on <b>October 16th-18th 2020</b>. It is a community-oriented conference for contributors and users of Nix and NixOS.<br />
```

Comment:
There is a typo on `years` :)

Reply:
you mean it should be `year's`?

Reply:
Only 'year' because it is singular

Reply:
Maybe you could try something like grammarly, even it isn't really privacy-friendly :/

Reply:
i want to say "the conference of this year" not "in this year, the conference"
maybe you misunderstand? but my english is not great and it is totally possible that i'm wrong

Reply:
I think that in both cases it should be "year" without a "s"

Reply:
i think that changes the meaning

this dictionaries also say it's "year's"
https://www.dict.cc/?s=diesj%C3%A4hrig&failed_kw=diesj%C3%A4hrige
https://dict.leo.org/german-english/diesj%C3%A4hrig
https://en.pons.com/translate/german-english/diesj%C3%A4hrig

Reply:
Honestly I think that you should write "This year NixCon will take place..." otherwise it's nonsense

Reply:
It feels wrong for be because if there is "years" that's plural but it's singular in the sentence and if it is "year's" then it's "This year is NixCon will take place" with two verbs so I think that's wrong too

Reply:
do you understand german?

i want to say
"Die diesjährige NixCon findet online statt"
instead of (your suggestion)
"Dieses Jahr findet die NixCon online statt"
i think that's a bit more elegant, at least in german

but let's wait for a third opinion

Reply:
Ok, I understand. Sorry for that "year's" is correct then